docs/resources/mws_vpc_endpoint.md

```diff
@@ -7,12 +7,14 @@ subcategory: "Deployment"
 
 -> **Note** This resource has an evolving API, which will change in the upcoming versions of the provider in order to simplify user experience.
 
-Enables you to register [aws_vpc_endpoint](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_endpoint) resources with Databricks such that they can be used as part of a [databricks_mws_networks](mws_networks.md) configuration.
+Enables you to register [aws_vpc_endpoint](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_endpoint) resources or gcp vpc_endpoint resources with Databricks such that they can be used as part of a [databricks_mws_networks](mws_networks.md) configuration.
```

Comment:
@nkvuong @nfx  Can I assume there will be a (https://registry.terraform.io/providers/hashicorp/gcp/latest/docs/resources/vpc_endpoint) after this PS is merged?

Reply:
GCP Terraform doc is missing this, so we have to point to this [open issue](https://github.com/hashicorp/terraform-provider-google/issues/11225) for now